gas.md

```diff
@@ -378,14 +378,30 @@ We assume that all operations interrogating the local state have complexity
   Save the local context (including the current register stack memory
   requirements), copy the arguments into the new context and jump to
   the call site.
+
+  We're assuming that, in an efficient implementation, saving the caller's
+  registers is a small constant time operation. One way or another, an
```

Comment:
This sounds like there is a single constant cost to save the entire collection of the caller's registers. I think we should only assume that there is a constant cost for each register (which is what the formula below says).